frontend/src/Pages/MatchLogs.tsx

```
@@ -0,0 +1,210 @@
+];
+
+const MatchLogs: React.FC = () => {
+  const getMatchDetails = (log: MatchLog) => {
+    const [player1, player2] = log.match.split(" vs ");
+    const stage = log.match.includes("First Round")
+      ? "First Round"
+      : log.match.includes("Semifinal")
+      ? "Semifinal"
+      : "Final";
+    let winner = "";
+    if (log.score && log.score.total) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)